apis/v1/gateway_types.go

```diff
@@ -923,6 +923,32 @@ type GatewayStatus struct {
 	// * "Programmed"
 	// * "Ready"
 	//
+	// Notes for implementors:
```

Comment:
so, just as the discussion raised on slack, can we add the tag `<gateway:util:excludeFromCRD>` here? :grimacing:

Reply:
Added.